### ISSUE TITLE
Incorrect chunkhash in jsonStats.hash

### ISSUE THREAD

**Issue by fresheneesz** [webpack/webpack#608]
I'm building a file with the `output.filename` "App.bundle.[chunkhash].js", and the filesystem is getting a build file named "App.bundle.f96d2a1a0885c2bff3c4.js". This is all fine, but when the bundle is built, its `jsonStats.hash` contains "6a0421c2b76dae3a3d2e". Isn't `jsonStats.hash` supposed to contain the same chunkhash the file is named with? 

I'm pretty sure this used to work, but in the newest version, it doesn't.

**Comment by bebraw** [comment]
@fresheneesz Is this relevant still?

**Comment by fresheneesz** [comment]
@bebraw What part specifically are you wondering about? Are you just asking if this can be closed?

**Comment by bebraw** [comment]
@fresheneesz Yeah. If there's a problem still, we can keep this open. Just trying to clean up the tracker a bit.

**Comment by fresheneesz** [comment]
@bebraw Ugh, I think there is more relevant work to do, but i'm finding it really hard to parse out here. The docs are rather gigantic and its hard to find this information in there. I'm not sure if it doesn't exist or is just hard to find. Also, my comment about having "devtools on something" is something i've forgotten how to verify or go back to. I guess you can close this until someone brings it up again. The main issue here is the API in this regard is a bit confusing.

**Comment by bebraw** [comment]
@fresheneesz Yeah. I've been working on something [more focused](http://survivejs.com/webpack/introduction/). If you have time at some point, please check that out. I'm curious to know what sort of topics would be worth covering.
